CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Conjur now raises a RoleNotFound when trying to authenticate a non-existing host in authn-k8s
+  [conjurinc/appliance#1546]https://github.com/conjurinc/appliance/issues/1546
```

Comment:
this should be `[conjurinc/appliance#1546](https://github.com/conjurinc/appliance/issues/1546`)` so it is linkable in a MD file

Reply:
this is not the correct issue. The issue you are fixing is that we now raise a NilError instead of an informative one. please add an issue if it's not present

Reply:
why are we linking to an issue filed in a private repo?

Reply:
nm - looks like this was updated later, and the changelog is current now.